### PR TITLE
edit pass

### PR DESCRIPTION
**Pull request by grahamgower**
---
I didn't get to the appendix, but everything else is looking great!

### CODE REVIEW

**Comment by grahamgower** [comment]
One other thing to note: I found the recombination and ARG sections the hardest parts to read. Possibly explaining the term "ancestral material" would help here (I find this term counter-intuitive).

File: paper.tex [hunk 1]
```diff
@@ -628,11 +637,11 @@ \subsection*{Simulating mutations}
 realistic analysis.
 
 There are many ways in which the mutation generation code
-in \msprime\ could be extended. For example, we intend to add support for
```
Comment (jeromekelleher) [inline paper.tex:631]: I'm not following the meaning of the changes here?

File: paper.tex [hunk 2]
```diff
@@ -675,8 +684,7 @@ \subsection*{Recombination}
 naive rejection sampling approach, and are somewhat slower
 to simulate than the exact coalescent with recombination. These
 models are therefore currently only appropriate for studying the
-SMC approximations themselves, although we intend to
-implement them more efficiently in future versions.
+SMC approximations themselves.
```
Comment (jeromekelleher) [inline paper.tex:687]: Good to say we intend to implement efficient approximation in the future?

File: paper.tex [hunk 3]
```diff
@@ -935,7 +944,7 @@ \subsection*{Gene conversion}
 
 An important avenue for future work
 will be to add further features to model bacterial
-evolution. In particular, we aim to include bacterial gene transfer and utilise
+evolution. In particular, to include bacterial gene transfer and utilise
```
Comment (jeromekelleher) [inline paper.tex:947]: Not sure removing the "we aim" improves clarity here

File: paper.tex [hunk 4]
```diff
@@ -1397,9 +1404,10 @@ \section*{Discussion}
 in shared infrastructure and mentorship in good software development practice.
 
 % THERE IS A BETTER WAY!!!
-Scientific software is vital apparatus, and must be engineered
+Scientific software is a vital apparatus, and must be engineered
```
Comment (jeromekelleher) [inline paper.tex:1407]: There's more than one bit of software, so I think the original is correct? https://www.cambridge.org/gb/academic/subjects/physics/general-and-classical-physics/building-scientific-apparatus-4th-edition?format=HB&isbn=9780521878586

**Review by jeromekelleher — COMMENTED** [review]
Looks great, minor grammatical comments.

File: paper.tex (hunk 1 above)
Comment (grahamgower) [reply paper.tex:631]: I changed the language in a few places to avoid commitments to future feature.

**Comment by grahamgower** [comment]
Ok, I've removed the changes regarding future commitments. IMO, it's a bad idea to make such commitments, because some things might be a lot of extra work for little gain (e.g. the SMC models), combined with possibly insufficient interest from users, and so things might never get implemented.

**Comment by jeromekelleher** [comment]
Well "intending" isn't the same thing as a commitment, it's just giving an indication of where we would like future development to go. We're definitely going to need approximations if we want to simulate Drospophila (which we do). The SMC should be the most extreme versions of these approximations.

**Pull request merged**